lib/cambiatus/commune/community.ex

```diff
@@ -50,11 +51,16 @@ defmodule Cambiatus.Commune.Community do
     field(:has_shop, :boolean, default: true)
     field(:has_kyc, :boolean, default: false)
 
+    # Social
+    field(:has_news, :boolean, default: false)
+
+    belongs_to(:highlighted_news, News)
```

Comment:
I like the comment but it should use spaces to scope better visually what's social. 

```suggestion
    # Social
    field(:has_news, :boolean, default: false)
    belongs_to(:highlighted_news, News)
    
```


Or even simpler

```suggestion
    field(:has_news, :boolean, default: false)

    belongs_to(:highlighted_news, News)
```